runtime/crab/src/pallets/fee_market.rs

```diff
@@ -27,8 +29,8 @@ where
 }
 
 frame_support::parameter_types! {
-	pub const FeeMarketPalletId: PalletId = PalletId(*b"da/feemk");
-	pub const FeeMarketLockId: LockIdentifier = *b"da/feelf";
+	pub const DarwiniaFeeMarketLockId: LockIdentifier = *b"da/feelf";
```

Comment:
If we change this, we need a migration for balance lock.

Reply:
The original `FeeMarketLockId` is not updated, just rename it here IMO. He removed the unused palllet_id.